docker-compose.yml

```diff
@@ -257,8 +331,67 @@ services:
         limits:
           memory: 256M
 
+  minio:
+    image: minio/minio:RELEASE.2024-06-26T01-06-18Z
+    networks:
+      nussknacker-iceberg-net:
+        aliases:
+          - warehouse.minio
+    hostname: minio
+    environment:
+      MINIO_ROOT_USER: admin
+      MINIO_ROOT_PASSWORD: password
+      MINIO_DOMAIN: minio
+      MINIO_REGION_NAME: us-east-1
+      MINIO_REGION: us-east-1
+    ports:
```

Comment:
But it will add another step to the "Setup preparation instruction".  Keeping the setup preparation instruction the shortest as possible is the key objective here :)

Reply:
yes, but at the same time, we risk a collision with already used user's ports. I see two possible solutions here:
1. we can map the ports to some, more uncommon ones (eg. 19000:9000)
2. in the instruction we can tell readers that the ports should be exposed to access minio & nessy consoles (and show that the easiest way to do so is using docker compose's port mapping) but in our solution, we can use additional nginx configs (we can mention that without deeper explaining)